cli/cmd/get.go

```diff
@@ -39,7 +58,7 @@ Only pod resources (aka pods, po) are supported.`,
 				return err
 			}
 
-			podNames, err := getPods(client)
+			podNames, err := getPods(client, options)
 			if err != nil {
 				return err
 			}
```

Comment:
This now has the potential to return no pods if there aren't any pods in the given namespace, in which case the command won't print anything. I think it would be better for us to print a message saying that no resources were found. That's how `conduit stat` and `kubectl get` handle it:

```
$ conduit stat pods
No traffic found.
```

```
$ kubectl get pods
No resources found.
```

For consistency with stat, you can just add this block of code on line 66 of this file:

```
			if len(podNames) == 0 {
				fmt.Fprintln(os.Stderr, "No resources found.")
				os.Exit(0)
			}
```